api/migrations/v20240517/projects.go

```diff
@@ -0,0 +1,87 @@
+package v20240517
```

Comment:
@danvixent, we don't need to make this backwards compatible.

Reply:
@subomi okay!